### DISCUSSION TITLE
Push notifications for all email notifications

### DISCUSSION THREAD

**Discussion by fischer-felix** [community/community#40230] [Mobile] [answered]
I have noticed that there are a lot more types of notifications you can receive via email than via push notifications.

For example, I cannot get push notifications for commits/issues/PRs on projects I have started.

Would be nice if that was possible.

**Comment by mattruggio** [comment, marked as answer]
Hey from the mobile team!  We have a few notification enhancement requests in our backlog that are similar and I will ensure one specifically mentions that example use case you suggested here: `get push notifications for commits/issues/PRs on projects I have started`.
